applications/MeshMovingApplication/python_scripts/ale_fluid_solver.py

```diff
@@ -3,64 +3,74 @@
 # Importing the Kratos Library
 import KratosMultiphysics
 KratosMultiphysics.CheckRegisteredApplications("MeshMovingApplication")
-import KratosMultiphysics.MeshMovingApplication
+import KratosMultiphysics.MeshMovingApplication as KratosMeshMoving
 
 # Other imports
 from python_solver import PythonSolver
 import python_solvers_wrapper_mesh_motion
 
 
 def CreateSolver(model, solver_settings, parallelism):
-    return ALEFluidSolver(model, solver_settings, parallelism)
+    return AleFluidSolver(model, solver_settings, parallelism)
```

Comment:
This class `AleFluidSolver` is not designed to be instantiated right ? (it does not have a fluid_solver) If yes then do we need a `CreateSolver` method here ?

Reply:
you are right, removed it

can you reapprove pls?